jitbuilder/control/Jit.cpp

```diff
@@ -35,13 +35,17 @@
 #include "runtime/CodeCache.hpp"
 #include "runtime/Runtime.hpp"
 #include "runtime/JBJitConfig.hpp"
+#include "runtime/RelocationRecord.hpp"
```

Comment:
Would you please separate the two JitBuilder related changes to a separate PR? I think we need to discuss these changes independently of the rest of the code you're contributing, and it doesn't make sense to pollute this review with a relatively independent set of changes.

Reply:
Thank you for the comment, the code will be shipped as a separate PR.